pkg/jobrunaggregator/jobrunaggregatorlib/gcs_client.go

```diff
@@ -42,6 +42,9 @@ func (o *ciGCSClient) ListJobRunNamesOlderThanFourHours(ctx context.Context, job
 	if err := query.SetAttrSelection([]string{"Name", "Created"}); err != nil {
 		return nil, nil, err
 	}
+
+	// When debugging, you can set the starting ID to a number such that you
+	// will process a relatively small number of jobsRuns.
```

Comment:
Nice idea adding these tips. If we could get these wired up to CLI options or env vars in the future that would be great.